modAionImpl/src/org/aion/zero/impl/forks/ForkUtility.java

```diff
@@ -130,6 +140,32 @@ public void enableNonceFork(long nonceForkBlockHeight) {
         this.nonceForkEnabled = true;
     }
 
+    // variable used by the signature swap fork
+    private boolean signatureSwapForkEnabled = false;
+    private long signatureSwapForkBlockHeight = Long.MAX_VALUE;
+
+    /**
+     * Returns a boolean value indicating if the signature swap fork is active for the given context (block
+     * number). We want the fork block itself to be a PoW block subject to the old pre-Unity rules,
+     * so we use a strict greater than comparison.
+     *
+     * @return {@code true} if the signature swap fork fork is active for the given context (block number), {@code
+     *     false} otherwise
+     */
+    public boolean isSignatureSwapForkActive(long contextBlockNumber) {
```

Comment:
Depending on the use case the second method checking equality may not be necessary and the first could check >=. Do we have a clear specification on when this fork becomes active?